framework/pom.xml

```diff
@@ -58,6 +58,7 @@
                   <outputFileName>tpcds_hivegenerated_parquet.tgz</outputFileName>
                   <unpack>true</unpack>
                   <outputDirectory>${project.basedir}/resources/Datasources</outputDirectory>
+                  <md5>b585a419fb5ceda6daa91a8bc19c2365</md5>
```

Comment:
It serves 2 purposes

1. Detect corrupt downloads. So we might as well add to all other files as well
2. I updated the tgz file on amazon. However the maven plugin did not use it and used one from its cache. So having this md5 forced maven to download the latest